spicepy/_client.py

```diff
@@ -1,11 +1,25 @@
 import os
 from pathlib import Path
+import platform
 import ssl
 import tempfile
 from typing import Union
 from urllib.request import urlretrieve
 
-from pyarrow import flight
+
+def is_m1() -> bool:
```

Comment:
```suggestion
def is_macos_arm64() -> bool:
```

I think its worth being precise here